test/test_sparse.py

```diff
@@ -385,6 +385,19 @@ def test_transpose_coalesce_invariant(self):
         self.assertTrue(x_coalesced.is_coalesced())
         self.assertFalse(y_uncoalesced.is_coalesced())
 
+    def test_t_empty(self):
+        x = self.SparseTensor(2, 3)
+        x.t_()
+        self.assertEqual(torch.Size([3, 2]), x.size())
+        self.assertEqual(0, x._indices().numel())
+        self.assertEqual(0, x._values().numel())
```

Comment:
Given the amount of difficulty we had thinking about what `x._dimI()` and `x._dimV()` should be, it would be good to assert that here (`self.assertEqual(x._dimI(), 2)`, `self.assertEqual(x._dimV(), 0)`)